### DISCUSSION TITLE
有汉字转换出来是方框

### DISCUSSION THREAD

**Discussion by Simon007-heiyewuxing** [note286/xduts#16] [Q&A] [answered]
\xdusetup的info中author中“珺”字打出来是方框，应该怎么办呢？
![image](https://user-images.githubusercontent.com/56477281/204247864-a9108f7d-443f-4df8-8cbb-8f3176a8ae1f.png)
![image](https://user-images.githubusercontent.com/56477281/204247922-4372c202-813c-4d8a-b584-87d2411833a1.png)

**Comment by note286** [comment, marked as answer]
请根据手册设置其他中文字体。

**Reply by Simon007-heiyewuxing** [reply]
太感谢您了，我把中文字体改成win就可以显示了，我试了一下，win字体和sinotype字体都可以显示“珺”，而其他三种字体都不行。
